airbyte-scheduler/persistence/src/main/java/io/airbyte/scheduler/persistence/DefaultJobPersistence.java

```diff
@@ -96,6 +96,9 @@
   private static final JSONFormat DB_JSON_FORMAT = new JSONFormat().recordFormat(RecordFormat.OBJECT);
   protected static final String DEFAULT_SCHEMA = "public";
   private static final String BACKUP_SCHEMA = "import_backup";
+  public static final String DEPLOYMENT_ID_KEY = "deployment_id";
+  public static final String METADATA_KEY_COL = "KEY";
```

Comment:
it was bugging that we were using the string literal all over the place.

Reply:
```suggestion
  public static final String METADATA_KEY_COL = "key";
```

I imagine the upcase was accidental? Feel like either this should be lower or the value should also be uppercase.

Reply:
i'm going based off of what's `initial.sql`. value is lower case and key is uppercase. in practice it doesn't matter. happy to change it.

Reply:
Oh weird. Guess this is more consistent then :p?

Reply:
yeah... though it's annoying because it gets serialized as lower case... so i guess i'll switch it after all..